setup.cfg

```diff
@@ -22,6 +22,8 @@ python_requires = >=3.8
 where = src
```

Comment:
The `setup.py`-less configuration is very cool.

Reply:
Yeah. And ultimately it's going to move into `pyproject.toml` as well (when `setuptools` adds support for that, which is their plan)